domains/ryan.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "ryan",
+
+    "owner": {
+        "email": "rg53734292@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "ryanthedev.vercel.app"
```

Comment:
We do not support vercel.

Reply:
But, why is there so many other names of people with vercel URLs?